src/index.ts

```diff
@@ -376,20 +383,22 @@ class DevServer {
     await this.startServer();
   }
 
-  async watch(startAdapter: boolean, noInstall: boolean): Promise<void> {
+  async watch(startAdapter: boolean, noInstall: boolean, doNotWatch: string | string[] | undefined): Promise<void> {
+    const doNotWatchArr = typeof doNotWatch === 'string' ? [doNotWatch] : doNotWatch || [];
```

Comment:
Maybe better out in an if clause it is a big ugly imo

Reply:
Yes ... then it would be a let variable just becaus ethe value is different by the if ... Because this is also such a topic if "const" is better if never changed afterwards ......... ;-)

Reply:
Will change